packages/react-components/react-motion-components-preview/library/src/components/Scale/Scale.ts

```diff
@@ -1,46 +1,64 @@
-import {
-  motionTokens,
-  PresenceMotionFn,
-  createPresenceComponent,
-  createPresenceComponentVariant,
-} from '@fluentui/react-motion';
+import { motionTokens, createPresenceComponent } from '@fluentui/react-motion';
+import { PresenceMotionFnCreator } from '../../types';
+import { opacityAtom, scaleAtom } from './ScaleAtoms';
+import { ScaleRuntimeParams_unstable, ScaleVariantParams_unstable } from './Scale.types';
 
 /** Define a presence motion for scale in/out */
-const scaleMotion: PresenceMotionFn<{ animateOpacity?: boolean }> = ({ animateOpacity = true }) => {
-  const fromOpacity = animateOpacity ? 0 : 1;
-  const toOpacity = 1;
-  const fromScale = 0.9; // Could be a custom param in the future
-  const toScale = 1;
-
-  const enterKeyframes = [
-    { opacity: fromOpacity, transform: `scale3d(${fromScale}, ${fromScale}, 1)`, visibility: 'visible' },
-    { opacity: toOpacity, transform: `scale3d(${toScale}, ${toScale}, 1)` },
-  ];
-
-  const exitKeyframes = [
-    { opacity: toOpacity, transform: `scale3d(${toScale}, ${toScale}, 1)` },
-    { opacity: fromOpacity, transform: `scale3d(${fromScale}, ${fromScale}, 1)`, visibility: 'hidden' },
-  ];
-
-  return {
-    enter: {
-      duration: motionTokens.durationGentle,
-      easing: motionTokens.curveDecelerateMax,
-      keyframes: enterKeyframes,
-    },
-    exit: { duration: motionTokens.durationNormal, easing: motionTokens.curveAccelerateMax, keyframes: exitKeyframes },
-  };
-};
+export const createScalePresence: PresenceMotionFnCreator<ScaleVariantParams_unstable, ScaleRuntimeParams_unstable> =
+  ({
+    enterDuration = motionTokens.durationGentle,
+    enterEasing = motionTokens.curveEasyEaseMax,
```

Comment:
```suggestion
    enterEasing = motionTokens.curveDecelerateMax,
```
Somehow this value was changed in the diff, but it hasn't changed in the [spec](https://microsoft.sharepoint-df.com/:fl:/g/contentstorage/CSP_654fdf19-4fed-43da-93ae-34db6738b27c/EYEv18MGSYhGnRCMc2_Pn0oB6CV4srJf7CPHA5lgYKI2ww?e=pBh1SV&nav=cz0lMkZjb250ZW50c3RvcmFnZSUyRkNTUF82NTRmZGYxOS00ZmVkLTQzZGEtOTNhZS0zNGRiNjczOGIyN2MmZD1iJTIxR2Q5UFplMVAya09UcmpUYlp6aXlmTjFPb1dxWXdGbEVrSEpTdVhDM2M0Zzl3ZG1hNXVQdlNMZ0JfamZXaW1RTiZmPTAxTTdJSzdZTUJGN0w0R0JTSlJCREoyRUVNT05YNDdIMksmYz0lMkYmYT1Mb29wQXBwJnA9JTQwZmx1aWR4JTJGbG9vcC1wYWdlLWNvbnRhaW5lciZ4PSU3QiUyMnclMjIlM0ElMjJUMFJUVUh4dGFXTnliM052Wm5RdWMyaGhjbVZ3YjJsdWRDMWtaaTVqYjIxOFlpRkhaRGxRV21VeFVESnJUMVJ5YWxSaVducHBlV1pPTVU5dlYzRlpkMFpzUld0SVNsTjFXRU16WXpSbk9YZGtiV0UxZFZCMlUweG5RbDlxWmxkcGJWRk9mREF4VFRkSlN6ZFpURE5YVmpZMFJVOUZWVFJTUTFvM1RFdFRTRE5ZUWs5TVVsQSUzRCUyMiUyQyUyMmklMjIlM0ElMjIxZmU5NTI2ZC01OWJjLTRjY2UtOGVlYS1kM2VlZjI1YTdiZjQlMjIlN0Q%3D). The Scale variants all use decelerate max for enter and accelerate max for exit.